common/djangoapps/third_party_auth/utils.py

```diff
@@ -30,6 +30,17 @@ def user_exists(details):
     return False
 
 
+def convert_slug_provider_id(provider):
```

Comment:
if this method specifically addresses the saml use case only (as opposed to say lti or others), shall we have the method name be sth like convert_saml_slug_provider_id  or so?